### ISSUE TITLE
How to change all dify templates to Chinese

### ISSUE THREAD

**Issue by kunlun122** [langgenius/dify#12210]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.14.2

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

 **I have read #8250**. and I have change the language to Chinese ,but the description of template is still english, .
![image](https://github.com/user-attachments/assets/3bc37f4c-0da5-4dba-8f4f-529a5c13af64)
![image](https://github.com/user-attachments/assets/88734b44-f697-4b10-90e0-2e569167576f)

### ✔️ Expected Behavior

I hope the template description can change according to the language settings

### ❌ Actual Behavior

the description of template is still english,

**Comment by kunlun122** [comment]
> I found a similar closed issue that might help you resolve the problem. The suggested solution is to ensure that the language codes for Chinese are included in the `web/i18n/languages.json` file and to set the default language in the `config/index.ts` file of your WebApp template. Here is a brief overview of the steps:
> 
> 1. **Update `web/i18n/languages.json`**: Make sure the language codes for Chinese ("zh-Hans" for Simplified Chinese or "zh-Hant" for Traditional Chinese) are included.
> 2. **Configure Default Language in `config/index.ts`**: Set the `default_language` to "zh-Hans" or "zh-Hant" in your application settings.
> 
> This should help change all templates to Chinese [[1]](https://github.com/langgenius/dify/issues/8250)[[2]](https://github.com/langgenius/dify/issues/8243).
> 
> _To continue talking to [Dosu](https://dosu.dev), mention @dosu._
> 
> Help Dosu learn! Give it feedback: [Great Response](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=great_response) | [Irrelevant Answer](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=irrelevant_answer) | [Incorrect Sources](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=incorrect_sources) | [Too Verbose](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=too_verbose) | [Hallucination](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=hallucination) | [Other](https://app.dosu.dev/response-feedback/e3122b98-6566-407d-b400-b385382e2d66?feedback_type=other)

I cann't find such files under Docker environment

**Comment by crazywoola** [comment]
You just need to switch to English and switch it back to Chinese.

**Comment by kunlun122** [comment]
@crazywoola I tried, but it does not work for me.

1.change to English
![image](https://github.com/user-attachments/assets/409fb7a8-af67-4c82-a89f-49d676d299ce)
![image](https://github.com/user-attachments/assets/c5de8aeb-fb94-42b6-b41b-c3f1ce63935b)

2.change back to Chinese,but the description of template is still english
![image](https://github.com/user-attachments/assets/e2e98477-ae18-4a4b-a465-cbf600864cda)
![image](https://github.com/user-attachments/assets/8398f06d-183d-467c-885f-b1f93b613f75)

**Comment by kunlun122** [comment]
I have just solved this problem. I checked the log of the API container and found that the API container tries to access tmpl.dify.ai to get templates. However, my server cannot access the Internet. I have set up a proxy for the container, and all templates changed to Chinese. 
![081SO}KJ$B66{H19EF`Z5CN](https://github.com/user-attachments/assets/29f2ec0c-6bed-42f5-b2a5-1bc2f451888a)

![image](https://github.com/user-attachments/assets/73127687-e64d-40fd-9433-4433359f2b4f)

**I hope the official team can pre-configure Chinese templates.**
